projects/storefrontapp/src/app/spartacus/features/added-to-cart-toast-feature.module.ts

```diff
@@ -0,0 +1,32 @@
+import { NgModule } from '@angular/core';
+import {
+  addedToCartToastTranslationChunksConfig,
+  addedToCartToastTranslations,
+} from '@spartacus/cart/added-to-cart-toast/assets';
+import {
+  AddedToCartToastRootModule,
+  ADDED_TO_CART_TOAST_FEATURE,
+} from '@spartacus/cart/added-to-cart-toast/root';
+import { provideConfig } from '@spartacus/core';
+
+@NgModule({
+  imports: [AddedToCartToastRootModule],
+  providers: [
+    provideConfig({
+      featureModules: {
+        [ADDED_TO_CART_TOAST_FEATURE]: {
+          module: () =>
+            import('@spartacus/cart/added-to-cart-toast').then(
+              (m) => m.AddedToCartToastModule
+            ),
+        },
+      },
+      i18n: {
+        resources: addedToCartToastTranslations,
+        chunks: addedToCartToastTranslationChunksConfig,
+        fallbackLang: 'en',
+      },
+    }),
+  ],
+})
+export class AddedToCartToastFeatureModule {}
```

Comment:
I don't think the cart toast should be a standalone feature.  Rather, it should be part of the cart base like the dialog.  Let's discuss.